tests/functional/FooTest.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+use Mado\QueryBundle\Action\CreateQuery;
+use Mado\QueryBundle\Interfaces\EntityClass;
+use Mado\QueryBundle\Queries\QueryBuilderFactory;
+use Mado\QueryBundle\Queries\QueryBuilderOptions;
+use Mado\QueryBundle\Repositories\BaseRepository;
+use PHPUnit\Framework\Testcase;
+
+final class FooTest extends Testcase
+{
+    public function setUp()
+    {
+        $this->manager = \Mado\QueryBundle\Common\Doctrine\EntityManager::getInstance();
+    }
+
+    public function testFilter()
+    {
+        $dql = new CreateQuery(
+            $this->manager,
+            QueryBuilderOptions::fromArray([
+                'select' => 'f',
+                'filters' => [
+                    'name|contains|1' => 'foo',
+                    'name|contains|2' => 'bar',
+                ]
+            ]),
+            new Foo()
+        );
+
+        $this->assertEquals(
+            "SELECT f0_.id AS id_0, f0_.name AS name_1 FROM Foo f0_ WHERE f0_.name LIKE ? AND f0_.name LIKE ?",
```

Comment:
What do you think to create an assert to compare filter array with what you pass into the request?
Same in many parts of the code

Reply:
There is no request here

Reply:
Yeah, but It's possible to ad an assert on filter array generated inside the QueryBundle?

Reply:
Could you explain better?